manifests/base/cuda-rstudio-buildconfig.yaml

```diff
@@ -118,11 +119,12 @@ spec:
       name: "cuda-rstudio-rhel9:latest"
   resources:
     limits:
-      cpu: "1500m"
-      memory: 8Gi
+      cpu: "2"
+      memory: 4Gi
     requests:
-      cpu: "1500m"
-      memory: 8Gi
+      cpu: "2"
+      memory: 4Gi
```

Comment:
Limit of a fractional number of cpu does not make much sense to me, I'd just give it 2, looking at the resources chart.